mainPage/index.html

```diff
@@ -255,7 +294,12 @@ <h2 id="whatdoesitbring" style="margin-bottom:10px">New Release EPICS Version 4.
 
        
 <p>To get started, see the <a href="https://sourceforge.net/projects/epics-pvdata/files/4.5.0/" >v4.5 Downloads</a> and the <a href="http://epics-pvdata.sourceforge.net/gettingStarted.html" > Getting Started with EPICS V4</a> guide.</p>
```

Comment:
The paragraph here on line 296 of index.html that starts "To get started, ..." still points to the 4.5.0 area, also needs fixing to point to 4.6.0.